web/models/organization_membership.ex

```diff
@@ -0,0 +1,41 @@
+defmodule CodeCorps.OrganizationMembership do
+  @moduledoc """
+  Represents a membership of a user in an organization.
+  """
+
+  use CodeCorps.Web, :model
+
+  alias CodeCorps.Organization
+  alias CodeCorps.User
+
+
+  schema "organization_memberships" do
+    field :role, :string
+
+    belongs_to :organization, Organization
+    belongs_to :member, User
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`, for creating a record.
+  """
+  def create_changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:member_id, :organization_id, :role])
+    |> validate_required([:member_id, :organization_id, :role])
+    |> assoc_constraint(:member)
+    |> assoc_constraint(:organization)
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`, for updating a record.
+  """
+  def update_changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:role])
+    |> validate_required([:role])
+  end
```

Comment:
I think we're missing this pattern from other models. We should apply there. Can we look and open issues for any that require it?